packages/vm/Cargo.toml

```diff
@@ -43,6 +43,7 @@ sha2 = "0.9.1"
 thiserror = "1.0"
 wasmer = { version = "1.0.1", default-features = false, features = ["jit", "singlepass"] }
 wasmer-middlewares = { version = "1.0.1" }
+k256 = { version = "0.7.2", features = ["ecdsa", "sha256"] }
```

Comment:
Do we really need the `sha256` feature? If I recall correctly this is used for `.verify` where you verify the original message instead of the hash.

Reply:
Not sure if it can be removed. Will check

Reply:
👍🏼 Moved it to `dev-dependencies`.